eventespresso/barista
# RTE with a11y

This PR creates a POC for an accessible RTE.

To test the editor:
- Checkout the branch - `gh pr checkout 596`
- `yarn; yarn sb`
- Open `New RTE` under `RichTextEditor` in Storybook

See #561 


![image](https://user-images.githubusercontent.com/18226415/104211810-8def6b80-545a-11eb-9dbf-934b73c19a14.png)
